rpm/389-ds-base.spec.in

```diff
@@ -364,6 +340,10 @@ Requires:         python%{python3_pkgversion}-lib389 = %{version}-%{release}
 A cockpit UI Plugin for configuring and administering the 389 Directory Server
 %endif
 
+%generate_buildrequires
+cd src/lib389
+%pyproject_buildrequires -g test
```

Comment:
As said elsewhere, this generates a BuildRequires on pytest... but is pytest used during the build?

Reply:
It is used for `%pyproject_check_import -e '*.test*'` which imports https://github.com/389ds/389-ds-base/blob/main/src/lib389/lib389/topologies.py and it needs `pytest`.

Actually, we might need to install it as a `lib389` dependency. @vashirov @bsimonova do I understand that we need to put `pytest` in the pyproject.toml's "dependencies" parameter?

Reply:
Added the comment! Thank you!
And yeah, we have the work planned for lib389 topologies and tests improvements. We'll consider the `lib389[topologies]` option. It sounds nice!